serverless/src/exportSearchClean/handler.js

```diff
@@ -0,0 +1,19 @@
+import { getDbConnection } from '../util/database/getDbConnection'
+
+/**
+ * Delete all rows for exports older than 30 days.  Policy automatically will delete objects from S3 bucket.
+ */
+const exportSearchClean = async (event, context) => {
+  // https://stackoverflow.com/questions/49347210/why-aws-lambda-keeps-timing-out-when-using-knex-js
+  // eslint-disable-next-line no-param-reassign
+  context.callbackWaitsForEmptyEventLoop = false
+
+  const dbConnection = await getDbConnection()
+
+  const threshold = new Date(new Date().setDate(new Date().getDate() - 30)).toISOString()
```

Comment:
`30` here should probably be settable from deployments, and should also be the same value that drives the bucket policy, but I don't understand how that works yet